eng/common/scripts/Verify-Links.ps1

```diff
@@ -201,11 +195,19 @@ function CheckLink ([System.Uri]$linkUri)
     }
   }
   
-  # Check if link uri includes locale info.
-  if ($checkLinkGuidance -and ($linkUri -match $locale)) {
-    LogWarning "DO NOT include locale $locale information in links: $linkUri."
-    $linkValid = $false
+  if ($checkLinkGuidance) {
+    # Check if the url is relative links
+    if (!$linkUri.IsAbsoluteUri) {
+      LogWarning "DO NOT use relative link $linkUri. Please use absolute link instead."
```

Comment:
We should setup some aka.ms links to the link guidance so people can read more if they hit this issue.